secant/Util/NavigationLinks.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+extension View {
+    func navigationLink<Destination: View>(
+        isActive: Binding<Bool>,
+        destination: @escaping () -> Destination
+    ) -> some View {
+        NavigationLink<Self, Destination>(
+            isActive: isActive,
+            destination: destination,
+            label: { self }
+        )
+    }
+
+    func navigationLinkEmpty<Destination: View>(
```

Comment:
I wouldn't think too much about this, what you have is fine here, but since this view is an Overlay, do you think navigationLinkOverlay makes more sense?

Reply:
Nice suggestion, I prefer that given the current implementation. I'll update it in subsequent PR. If we stumble across a more suitable way of doing it we can update accourdingly.